README.md

```diff
@@ -1,7 +1,228 @@
-Server and app host support for taco-simulate.
+Simulates your Apache Cordova application in the browser.
 
 # Installation
 
 ```
-npm install taco-simulator-server
+npm install -g cordova-simulate
 ```
+
+
+# Usage
+
+## API
+Use `require('cordova-simulate')` to launch a simulation via the API:
+
+```JavaScript
+var simulate = require('cordova-simulate');
+simulate(opts);
+```
+
+Where opts is an object with the following properties (all optional):
+
+* **platform** - any Cordova platform that has been added to your project. Defaults to `browser`.
+* **target** - the name of the browser to launch your app in. Can be any of the following: `chrome`, `chromium`, `edge`, `firefox`, `ie`, `opera`, `safari`. Defaults to `chrome`.
+* **port** - the desired port for the server to use. Defaults to `8000`.
+* **dir** - the directory to launch from (where it should look for a Cordova project). Defaults to cwd.
+* **simhostui** - the directory containing the UI specific files of the simulation host. Defaults to the bundled simulation host files, found in `src/sim-host`.
```

Comment:
My previous comment got hidden because of "commenting on a previous diff", so I'm adding it here:

I would also update the description: "... Defaults to the bundled simulation host UI files, found in src/sim-host/ui."